rasa/core/tracker_store.py

```diff
@@ -186,6 +186,8 @@ def _deserialise_dialogue_from_pickle(
         sender_id: Text, serialised_tracker: bytes
     ) -> Dialogue:
 
+        # TODO: not sure if we should remove because it says "it will be deprecated in 2.0"
+        #       instead of removed.
         logger.warning(
```

Comment:
This should be turned into one of your new warnings now, no?

Reply:
yeah, I left it because of the `TODO:`, not sure if it's ok to remove it. If we do not remove it, I'll turn it into the new syntax

Reply:
How i interpreted it: 

someone during 1.x decided to deprecate this. But instead of saying "this will be removed in 2.0" they said "this will be deprecated in 2.0" 

Someone upon 2.0 cleaned up the deprecation warnings. But they were unsure whether they could remove this functionality because it didn't say it would be removed in 2.0 

IMO we should deprecate this in 2.0, to be removed in 3.0. Which is what leaving the warning does -- was just saying that we should update it to your new format :)

Reply:
ok, will got with this then